viewer/index.html

```diff
@@ -20,8 +20,7 @@
     <link rel="stylesheet" type="text/css" href="css/jquery-ui.min.css" />
     <script src="js/arrayFind.js"></script>
     <script src="js/string.polyfill.js"></script>
-    <script src="js/knockout-latest.js"></script>
-
+    <script src="https://knockoutjs.com/downloads/knockout-3.4.2.debug.js"></script>
```

Comment:
This change  is result of merge conflict resolved with Thematic Maps branch, but should have been estMap. Need to revert.